pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+{package} {version} {:snowflake:}
+
+**Destination channel:** {Snowflake | defaults}
+
+### Links
+
+- [{ticket_number}]() 
+- [Upstream]()
+- [conda-forge]()
+- [pypi]()
+- Dependencies:
+  - ...
```

Comment:
What's the purpose of this? Can you add a comment into the template, too?

Reply:
It's intended for links to PRs for recently built dependencies as context. This is particularly useful when some of those dependencies are still on staging channels. I'll add a comment for clarity.

@Jrice1317 Definitely not linking/listing *all* dependencies since that'd just be duplicating the recipe.